docs/notebooks/tutorial.ipynb

```diff
@@ -204,30 +223,35 @@
    "execution_count": 3,
    "metadata": {
     "collapsed": false,
+    "deletable": true,
+    "editable": true,
     "scrolled": false
    },
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "[2017-04-24 18:18:46.912285] INFO: Performance: after split: sid: Equity(0 [AAPL]), amount: 2340.0, cost_basis: 55.65, last_sale_price: 101.25\n",
-      "[2017-04-24 18:18:46.912451] INFO: Performance: returning cash: 0.0\n",
-      "[2017-04-24 18:18:54.383697] INFO: Performance: after split: sid: Equity(0 [AAPL]), amount: 28220.0, cost_basis: 15.77, last_sale_price: 88.99\n",
-      "[2017-04-24 18:18:54.383934] INFO: Performance: returning cash: 0.0\n",
-      "[2017-04-24 18:19:08.603957] INFO: Performance: Simulated 3521 trading days out of 3521.\n",
-      "[2017-04-24 18:19:08.604134] INFO: Performance: first open: 2000-01-03 14:31:00+00:00\n",
-      "[2017-04-24 18:19:08.604221] INFO: Performance: last close: 2013-12-31 21:00:00+00:00\n"
+      "[2017-06-15 15:20:16.013533] WARNING: Loader: Refusing to download new benchmark data because a download succeeded at 2017-06-15 15:07:41+00:00.\n",
```

Comment:
I believe it means that we had ran a download at `2017-06-15`, and because we're not requesting any data after that date, we're not going to download any new data (because we don't need any new data)